.ci/Jenkinsfile

```diff
@@ -146,7 +151,11 @@ pipeline {
     }
     stage('Release') {
       agent { label 'linux && immutable' }
-      options { skipDefaultCheckout() }
+      options {
+        skipDefaultCheckout()
+        timeout(time: 12, unit: 'HOURS')
+        warnError('The Release was not make')
```

Comment:
aha!! That's then the way to avoid the abort when waiting for an input  :)

Reply:
the build will be marked as unstable, let's see how it works if it is too annoying, we would try to find another way to mark only the stage as unstable, maybe something like that it is odd but I do not want to touch the build result manually. Also, we can make a PR to add `buildResult` and `stageResult` to warnError, I will do it `ignoreError` :)

```
post {
  unsuccessful {
    catchError(buildResult: 'SUCCESS', message: 'msg', stageResult: 'UNSTABLE') {
      sh(label: 'Failure', script: 'exit 1')
    }
  }
}
```